main/curl/APKBUILD

```diff
@@ -101,12 +104,11 @@ build() {
 }
 
 check() {
-	cd "$builddir"
-	make -C tests nonflaky-test
+	cd "$builddir"/tests
```

Comment:
This change does not improve a lot and only introduces noise. I would leave it out.